specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/subscriptions.json

```diff
@@ -147,6 +147,73 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Billing/enrollmentAccounts/{enrollmentAccountName}/providers/Microsoft.Subscription/subscription/{subscriptionId}/createSubscription": {
+      "put": {
+        "description": "Creates an Azure subscription",
+        "operationId": "SubscriptionFactory_CreateSubscriptionWithSubId",
+        "x-ms-examples": {
+          "createSubscription": {
+            "$ref": "./examples/createSubscription.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionCreationResult"
+            }
+          },
+          "202": {
```

Comment:
The 202 pattern is fine, but generally 201 is preferred for resource create. https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations